src/oid.rs

```diff
@@ -86,6 +92,8 @@ lazy_static! {
     pub static ref KcdsaSha224: ObjectIdentifier = vec![1, 2, 410, 200004, 1, 40].into();
     pub static ref KcdsaSha256: ObjectIdentifier = vec![1, 2, 410, 200004, 1, 27].into();
     pub static ref EcKcdsa:     ObjectIdentifier = vec![1, 0, 14888, 3, 0, 5].into();
+
+    pub static ref keyAttestationAuthorityRole: ObjectIdentifier = vec![1, 3, 6, 1, 4, 1, 49690, 7, 1].into();
```

Comment:
Do we have any other Fortanix-defined OIDs in this crate?

Reply:
No.
Should I not include it in this crate?
I was thinking it's fine to be public since it's  part of attestation certificates

Reply:
I would mildly argue for not putting it in this crate if there are no other Fortanix-specific OIDs

Reply:
Yeah, it's not necessary to put it here

Reply:
removed